data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1856 @@
+			`	"True it is, that spreading around posters, banners and flyers as propaganda, we have. Only, not for some movie or other media they were. Jabs at the Heliarchy, they all have been, means of getting the people thinking, so that maybe recruit them, we can. Help greatly with that, what we secured from <origin> will."`
+			label question
+			choice
+				`	"What did you bring in those crates? Weapons?"`
+				`	"Who are you?"`
+					goto who
+			`	"Evidence. Proof. Of their crimes unpunished, of treason against one's own. Hold countless billions, the Kimek factory worlds do, and grounds for many of the Heliarchs' senseless arrests, they are. Write off disappearances as work accidents they do, but records, they keep, and tracking the captured 'criminals' they always are. Our target, those records were. There described, the whereabouts or fates of many are."`
+			label who
+			`	He remains quiet for a while, focusing his eyes on the images, then on yours as he hands you one of the posters. "Called the Lunarium Redoubt, my group is. Nickname us rebels and outlaws, the Heliarchs kindly do in their broadcasts. Helped us you have, <last>, and help us a lot more, in other ways you can." He pauses, still locking eyes with you. "Think not the Heliarchs as benefactors. Ruin us they will, if stop them we cannot."`
+				goto end
+			label resistance
+			`	"Self explanatory, I imagine?" he says before he picks up the last few of them from the box. "Spread these around the planets, we of course will, but more valuable, the information in the unopened boxes and crates is."`
+			choice
+				`	"What information?"`
+					goto information
+				`	"Is that why you were in such a rush to leave before?"`
+			`	"Hoped we had, that suffice as a distraction the skywriting and rain of flyers would, so that safe for the others to break in and steal the data it was," he explains, sighing. "Mistaken, we were. Set their patrols off all around, our previous propaganda efforts apparently did. Go into hiding for a while, they'll need to."`
+				goto end
+			label information
+			`	"Documents, recorded data of the fates or whereabouts of many who 'disappeared,' we stole. A common thing it is, on Kimek factory worlds, for Heliarchs to write them off as work accidents. Investigate the data thoroughly we will, and maybe uncover the coordinates of their prison - or morgue - we finally can."`
+			label end
+			`	He thanks you for helping, and leaves your ship with the remaining posters. You go to check the spaceport and portions of the city later, but find neither Tummug, his colleagues, or any of the posters.`
+
+
+
+mission "Resistance: Combat Training 1"
+	name "Bootcamp"
+	description "Bring <bunks> Lunarium members, along with their <cargo> to <destination>, where they will purchase some ships and use them to fight pirates to train their combat tactics."
+	minor
+	passengers 54
+	cargo "supplies" 9
+	source
+		attributes arach
+	destination "Zug"
+	to offer
+		random < 20
+		or
```

Comment:
The idea is for this to offer when you've completed at least one of [Propaganda, Evacuation, Smuggling], meaning you need to have met a Lunarium leader and learned somewhat about them, finally getting the increment to the assisted resistance counter.
House Bebliss' chain isn't in because you could complete that entirely without ever knowing of the Lunarium, as it doesn't rely on any knowledge of them and doesn't reveal any if you didn't know already.
The supplies you mean the other smuggling missions? If so I just picked the last one to keep with the other ones being the ones with the increment.